test/models/category_test.rb

```diff
@@ -1,7 +1,49 @@
 require "test_helper"
 
 class CategoryTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    DatabaseCleaner.start
+    site = create(:site)
+    @categories = create_list(:category, 5, site: site)
```

Comment:
`factory :category` uses sequence for category name.
So all categories have different name each other.

On the other hand, we may need test that some categories have same name.